content/en/news/releases/1.6.x/announcing-1.6/change-notes/index.md

```diff
@@ -0,0 +1,48 @@
+- SDS for mixer? Not sure we need this or its an implementation detail
+- ***Improved*** Prometheus integration experience by adding standard Prometheus scrape annotations to proxies and the control plane workloads. This removes the need for specialized configuration to discover and consume Istio metrics. More details are available in the [operations guide for Prometheus](https://preliminary.istio.io/docs/ops/integrations/prometheus#option-2-metrics-merging).
+- ***Updated*** default Telemetry V2 configuration to avoid using host header to extract destination service name at gateway. This prevents unbound cardinality due to untrusted host header and implies that destination service labels are going to be omitted for request hits Blackhole and Passthrough at gateway.
+- ***Improved*** support for [customizing v2 metrics generation](https://istio.io/docs/tasks/observability/metrics/customize-metrics/). This allows mesh operators to add and remove labels used in Istio metrics based on expressions over the set of available request and response attributes.
+- ***Improved*** the output of the istioctl install command.
+- ***Added*** a new command, istioctl install. This is an alias for manifest apply.
+
+## Installation Impacts
+- TODO: add a statement around file mounted gateway removed and impact to users.
+- ***Removed*** the legacy Helm charts. Please see the [Upgrade guide](https://preliminary.istio.io/docs/setup/upgrade/) for migration.
+- ***Removed*** most configuration flags and environment variables for the proxy. These are now read directly from the mesh configuration.
+- ***Changed*** the proxy readiness probe to port 15021
+- ***Removed*** Security alpha API
+- ***Removed*** the Citadel, Sidecar Injector, and Galley deployments. These were disabled by default in 1.5, and all functionality has moved into Istiod.
+- ***Removed*** ports 15029-15032 from the default ingressgateway. It is recommended to expose telemetry addons by [Host routing](https://preliminary.istio.io/docs/tasks/observability/gateways/) instead.
+- TODO helm3/kustomize support? should we mention something here?
+- ***Improved*** installation to not manage the installation namespace, allowing more flexibility.
+- ***Removed*** the legacy istio-pilot configurations, such as Service.
+- ***Removed*** built in Istio configurations from the installation, including the Gateway, VirtualServices, and mTLS settings.
+- ***Added*** a new preview profile, allowing users to try out new experimental features.
+- ***Added*** `istioctl install` command as a replacement for `istioctl manifest apply`.
+- ***Added*** functionality to save installation state in a CustomResource in the cluster.
+- ***Changed*** Gateway ports used (15020) and [resolution](https://github.com/istio/istio/pull/23432#issuecomment-622208734) for end users
+- ***Added*** Allow users to add a custom hostname for istiod
+
+## Bug fixes / Misc
```

Comment:
`## Bug Fixes & miscellaneous changes`

Misc is not a word in our spelling file (nor should be 👍

Reply:
Thanks. Done.